Resources/Textures/_DV/Objects/Devices/communication.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Inspired by the cheese wedge sprite from tg station https://github.com/tgstation/tgstation/blob/master/icons/obj/food/food.dmi modified by @ps3moira",
+    "copyright": "Cheese wedge sprite inspired by tgstation. Syndicat sprite by Aleksh, yarn radio by @Stxcking.",
```

Re-add the attribution for the cheese wedge sprite.

```suggestion
    "copyright": "Inspired by the cheese wedge sprite from tg station https://github.com/tgstation/tgstation/blob/master/icons/obj/food/food.dmi modified by @ps3moira. Syndicat sprite by Aleksh, yarn radio by @Stxcking.",
```